src/index.js

```diff
@@ -0,0 +1,49 @@
+import _ from 'lodash';
+import './style.css';
+import printMe from './print.js';
+const data=[
+    {
+       description:"Todo listings",
+       "completed":true,
+       "index":1
+    },
+    {
+        description:"Todo listings",
+        "completed":true,
+        "index":1
+     },
+     {
+        description:"Todo listings",
+        "completed":true,
+        "index":1
+     },
+]
+const ul=document.createElement('ul')
+const div=document.createElement('div')
+div.innerHTML=`<h3>Today's todo</h3>`
+ul.appendChild(div)
+const input=document.createElement('div')
+input.innerHTML=`<input type="text" class="enter" placeholder="Enter something" />`
+ul.appendChild(input)
+
+const addliststodom=()=>{ 
+
+data.map((item)=>{
+   const li=document.createElement('li')
+   li.innerHTML=`
+   <div class="delete">
+   <div class="flexer"><input type="checkbox" class="check"/><h5>${item.description}</h5></div>
+   <h6>del</h6>
+   </div>
+   `
+
+ul.append(li)
+})
+const item=document.querySelector(".container")
+item.appendChild(ul)
+}
```

Comment:
- [ ] It is required to style your project like the design given. Kindly look into the design properly and work your design alongside the given design.

### Your Design
![your-design](https://user-images.githubusercontent.com/26597432/202802124-426a2f21-526d-4bed-b24b-82b58b10f4e2.png)

### Guiding Template
![minimalist](https://user-images.githubusercontent.com/26597432/202803049-b7c34e05-6b04-45a1-9799-e3040f8da961.png)